spec/traits.dd

```diff
@@ -1127,6 +1128,22 @@ void main()
 ---
 )
 
+$(H2 $(GNAME getCppNamespaces))
+	$(P The argument is a symbol.
+	The result is a tuple of strings, possibly empty, that correspond to the namespaces the symbol resides in.
+	)
+
+$(SPEC_RUNNABLE_EXAMPLE_COMPILE
+---
+extern(C++, "ns")
```

Comment:
SPEC_RUNNABLE_EXAMPLE_COMPILE insets a `void main(){}` around this code block resulting in `namespaces are not allowed here`, which macro should I use instead?

Reply:
Test also with a non-string namespace:
```d
extern(C++, ns2) void fun();
```

Reply:
Actually, if this is just for the spec, just make sure the unit tests for the implementation test with string and non-string namespace.